common/changes/@cadl-lang/compiler/folding_2022-07-05-16-01.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@cadl-lang/compiler",
+      "comment": "implementation of folding",
```

Comment:
This comment goes out to customers in the change log. Try to word it for that audience. Something like "Improve folding in IDE."